docs/source/entities.mdx

```diff
@@ -0,0 +1,321 @@
+---
+title: Entities in Apollo Federation
+sidebar_title: Entities (basics)
+description: Resolve types across multiple subgraphs
+---
+
+import { CodeColumns, ExpansionPanel } from 'gatsby-theme-apollo-docs';
+
+> ⚠️ Some details of entity behavior have changed in Federation 2. For a summary of these changes, [see what's new](/federation-2/new-in-federation-2/).
+
+In a federated graph, an **entity** is an object type that defines its fields across multiple subgraphs. Each subgraph is responsible for resolving whichever fields it defines for an entity.
+
+Here, the `Product` entity's fields are defined across two subgraphs:
+
+<CodeColumns>
+
+```graphql
+# Products subgraph
+type Product @key(fields: "id") {
+  id: ID!
+  name: String!
+  price: Int
+}
+```
+
+```graphql
+# Inventory subgraph
+type Product @key(fields: "id") {
+  id: ID!
+  inStock: Boolean!
+}
+```
+
+</CodeColumns>
+
+Types _besides_ object types (unions, interfaces, etc.) _cannot_ be entities.
+
+## Defining an entity
+
+In a subgraph schema, you can designate any existing object type as an entity by adding the `@key` directive to its definition, like so:
+
+```graphql{2}
+# Products subgraph
+type Product @key(fields: "id") {
+  id: ID!
+  name: String!
+  price: Int
+}
+```
+
+The `@key` directive defines the entity's **primary key**, which consists of one or more of the type's `fields`. In this example, the `Product` entity's primary key is its `id` field.
```

Comment:
I'd be in favour, at this point, to mention that every time you write a `@key` directive, then you will have to write a corresponding "reference resolver" and link to the proper section. In many ways, all a `@key` is doing as far as the implementation is concerned is declaring that a reference resolver for that key will exist at runtime and so the query planner can use it, so I think creating that strong link between a key and it's resolver early is useful and a good idea.

Reply:
This section now describes the process of creating an entity as a two-step process, first adding a `@key`, then defining a reference resolver. So the connection is much stronger!